testing/py3-rdflib/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Maintainer: Luca Weiss <luca@z3ntu.xyz>
+pkgname=py3-rdflib
+_pyname=rdflib
+pkgver=4.2.2
+pkgrel=0
+pkgdesc="A Python library for working with RDF, a simple yet powerful language for representing information"
+url="https://rdflib.readthedocs.org/"
+arch=noarch
+license='BSD-3-Clause'
+depends="py3-isodate py3-six"
+makedepends="py3-setuptools"
+_pypiprefix="${_pyname%${_pyname#?}}"
+source="https://files.pythonhosted.org/packages/source/$_pypiprefix/$_pyname/$_pyname-$pkgver.tar.gz"
+builddir="$srcdir/$_pyname-$pkgver"
+options="!check"
```

Comment:
please add reason

Reply:
I've got the tests running now, it was not easy because of partly old python3-incompatible releases and circular dependencies.